sdk/src/wormhole.ts

```diff
@@ -190,7 +196,7 @@ export class WormholeContext extends MultiProvider<Domain> {
     recipientAddress: string,
     relayerFee?: string,
     payload?: any,
-  ): Promise<ContractReceipt | Transaction> {
+  ): Promise<ContractReceipt | Transaction | TransactionBlock> {
```

Comment:
Really should implement the type parameters, like Martin showed us in the wallet aggregator sdk